# name: traceback.print_exce 
traceback.print_exception(exc, /, [value, tb, ]limit=None, file=None, chain=True)